api/src/main/java/marquez/api/SearchResource.java

```diff
@@ -41,9 +49,12 @@ public class SearchResource {
   private static final String DEFAULT_LIMIT = "10";
   private static final int MIN_LIMIT = 0;
 
+  private final SearchService searchService;
```

Comment:
Given that the new search API is experimantl, do we want to prefix the `job` and `dataset` endpoints with `v2beta/search`?